guides/source/generators.md

```diff
@@ -219,15 +219,14 @@ If we want to avoid generating the default `app/assets/stylesheets/scaffolds.scs
   end
 ```
 
-The next customization on the workflow will be to stop generating stylesheet, JavaScript, and test fixture files for scaffolds altogether. We can achieve that by changing our configuration to the following:
+The next customization on the workflow will be to stop generating stylesheet, and test fixture files for scaffolds altogether. We can achieve that by changing our configuration to the following:
```

Comment:
Can we remove the `,` in `stop generating stylesheet, and test fixture files`? I think it reads better without it.

Reply:
I'm not sure, since we use "Oxford comma" in the guides https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#oxford-comma, (but probably we can remove that comma here since we need 3 or more terms guided by https://en.wikipedia.org/wiki/Serial_comma)

Reply:
Yes, I believe oxford commas are necessary only when listing 3+ items in a sentence. I wouldn't put a comma in-between listing 2 items because it sounds unnatural.

Reply:
Agreed. Fixed.